db/migrate/20220109135029_add_rooms_to_floor.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoomsToFloor < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :rooms, :floor, null: true, foreign_key: true
```

Comment:
```suggestion
    add_reference :rooms, :floor, null: true, foreign_key: true
```
Would alter it to null: false (via new migration tho)